c/public/lib/source/ttread/ttread.c

```diff
@@ -2066,23 +2853,35 @@ static void glyfReadCompound(ttrCtx h, GID gid, GID *mtx_gid, int depth) {
         saveoff = srcTell(h);
 
         /* Read component glyph */
-        if (h->glyphs.array[component].info.sup.begin == ABF_UNSET_INT)
-            break; /* Missing component glyph! */
-
-        nContours = glyfReadHdr(h, component);
-        if (nContours >= 0) {
+        if (h->glyphs.array[component].info.sup.begin == ABF_UNSET_INT) {
+            nContours = 0;  /* Empty component glyph */
```

Comment:
The value stored in `nContours` here is never read.

Reply:
correct. removed line